exotic.py

```diff
@@ -1314,11 +1368,12 @@ def parse_args():
                     print("\nCannot find " + targetName + " in the NASA Exoplanet Archive. Check file: eaConf.json")
                     targetName = str(input("Enter the Planet Name Again or type 'manual': "))
                     if targetName == 'manual':
-                        CandidatePlanetBool = True
+                        CandidatePlanetBool = False
                         break
-            idx = planets.index(targetName.lower().replace(' ', '').replace('-', ''))
-            pDict = new_getParams(data[idx])
-            print('\nSuccessfuly found ' + targetName + ' in the NASA Exoplanet Archive!')
+            if CandidatePlanetBool:
+                idx = planets.index(targetName.lower().replace(' ', '').replace('-', ''))
+                pDict = new_getParams(data[idx])
+                print('\nSuccessfuly found ' + targetName + ' in the NASA Exoplanet Archive!')
```

Comment:
issue #96

Reply:
I mixed up the True and False for candidate planets. Will revert back once updated with new planetary suggestions.

Reply:
I think you might have mixed up what a `CandidatePlanet` is - this is a non-confirmed planet, so it is still a "candidate". So if `targetName == 'manual'` I think `CandidatePlanetBool = True` instead. Correct?

Reply:
Went ahead and fixed it and waiting on your opinion of planet checking.